Formula/sqlite.rb

```diff
@@ -29,14 +29,11 @@ def install
     # Default value of MAX_VARIABLE_NUMBER is 999 which is too low for many
     # applications. Set to 250000 (Same value used in Debian and Ubuntu).
     ENV.append "CPPFLAGS", "-DSQLITE_MAX_VARIABLE_NUMBER=250000"
-    ENV.append "CPPFLAGS", "-DSQLITE_ENABLE_RTREE=1"
```

Comment:
It seems this and `SQLITE_ENABLE_JSON1` are not on by default? Or have I misread the following:

https://sqlite.org/compile.html#_options_to_enable_features_normally_turned_off

Reply:
It's enabled by default in the configure script.

Reply:
So why does the website say they are normally turned off? Does it hurt to keep them, even if they are already on?

Reply:
I'm not sure what they mean 'Normally Turned Off', but one of the main feature of 3.35 is math functions and `SQLITE_ENABLE_MATH_FUNCTIONS` is in this section and it's enabled by default in configure.